Features/Stories/Fieldtypes/AuthorsFieldType.feature

```diff
@@ -0,0 +1,180 @@
+        And I add another author
+        And I set "Ana" as the "name" Field Value of the "second" author
+        And I set "ana@acme.com" as the "email" Field Value of the "second" author
+        And I publish the content
+        Then the Content is successfully published
+
+    @javascript
+    Scenario: Publishing an authors Field fails validation when using an invalid email
+        Given a Content Type with an "authors" Field exists
+        When I create a content of this Content Type
+        And I set "Paul" as the "name" Field Value of the "first" author
+        And I set "paul" as the "email" Field Value of the "first" author
+        And I publish the content
+        Then the Content is not published
+        And I see a "!" before the "email" Field Value
+
+    @javascript
+    Scenario: Publishing an required authors Field fails validation when using an empty author
+        Given a Content Type with a "requited" "authors" exists
+        When I create a content of this Content Type
+        And I set an empty value as the "name" Field Value of the "first" author
+        And I set an empty value as the "email" Field Value of the "first" author
+        And I publish the content
+        Then Publishing fails
+        And I see a "This field is required" before the "name" Field Value
+        And I see a "This field is required" before the "email" Field Value
+
+    ##
+    # Update Content using a Content Type that has an Authors Fieldtype
+    ##
+    @javascript
+    Scenario: Updating an author Field works
+        Given a Content Type with an "authors" Field exists
+        And a Content of this Content Type exists
+        When I update this content setting "Paul" as the "name" Field Value of the "first" author
+        And I update this content setting "paul@acme.com" as the "email" Field Value of the "first" author
+        And I publish the content
+        Then the Content is successfully published
+
+    @javascript
+    Scenario: Updating an authors Field fails validation when using an invalid email
+        Given a Content Type with an "authors" Field exists
+        And a Content of this Content Type exists
+        When I set "paul" as the "email" Field Value of the "first" author
+        And I publish the content
+        Then the Content is not published
+        And I see a "!" before the "email" Field Value
+
+    @javascript
+    Scenario: Publishing a required authors Field fails validation when using an empty author
+        Given a Content Type with a "requited" "authors" Field exists
+        And a Content of this Content Type exists
+        When I set an empty value as the "name" Field Value of the "first" author
+        And I set an empty value as the "email" Field Value of the "first" author
+        And I publish the content
+        Then the Content is not published
+        And I see a "!" before the "name" Field Value

```

Comment:
Sorry. I explained myself wrong. I'll replace the sentence by 
"And I see a "This field is required" before the "name" Field Value". Like this, we not only validate that the message appears, as we validate the message text.